apps/api/src/app/shared/framework/types/utils.types.ts

```diff
@@ -0,0 +1,63 @@
+/* cSpell:enableCompoundWords */
```

Comment:
Check out the Typescript typing tests in `utils.types.spec.ts` to see these in action.